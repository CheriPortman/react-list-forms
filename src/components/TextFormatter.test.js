import React from 'react';
import { shallow } from 'enzyme';
import TextFormatter from './TextFormatter';

describe('TextFormatter component', () => {
  it('renders the text', () => {
    const wrapper = shallow(<TextFormatter />);
    expect(wrapper).toMatchSnapshot();
  });
});
