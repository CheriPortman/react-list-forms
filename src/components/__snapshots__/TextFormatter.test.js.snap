// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextFormatter component renders the text 1`] = `
<Fragment>
  <select
    name="font"
    onChange={[Function]}
    value="Banner"
  >
    <option
      key="Avatar"
      value="Avatar"
    >
      Avatar
    </option>
    <option
      key="Banner"
      value="Banner"
    >
      Banner
    </option>
    <option
      key="Alligator"
      value="Alligator"
    >
      Alligator
    </option>
    <option
      key="Linux"
      value="Linux"
    >
      Linux
    </option>
    <option
      key="Acrobatic"
      value="Acrobatic"
    >
      Acrobatic
    </option>
    <option
      key="Barbwire"
      value="Barbwire"
    >
      Barbwire
    </option>
    <option
      key="Doom"
      value="Doom"
    >
      Doom
    </option>
  </select>
  <input
    name="text"
    onChange={[Function]}
    value=""
  />
  <input
    name="color"
    onChange={[Function]}
    type="color"
    value="#FF4500"
  />
  <pre
    style={
      Object {
        "color": "#FF4500",
      }
    }
  />
</Fragment>
`;
